src/Microsoft.NetCore.Analyzers/Core/Security/SystemSecurityCryptographyResources.resx

```diff
@@ -261,6 +261,15 @@
   <data name="SetViewStateUserKeyMessage" xml:space="preserve">
     <value>The subclass {0} of Page does not set the ViewStateUserKey property in method OnInit</value>
   </data>
+  <data name="UseSharedAccessProtocolHttpsOnly" xml:space="preserve">
+    <value>Use SharedAccessProtocol HttpsOnly</value>
+  </data>
+  <data name="UseSharedAccessProtocolHttpsOnlyDescription" xml:space="preserve">
+    <value>HTTPS encrypts network traffic. Use HttpsOnly, rather than HttpOrHttps, to ensure network traffic is always encrypted to help prevent disclosure of sensitive data.</value>
+  </data>
+  <data name="UseSharedAccessProtocolHttpsOnlyMessage" xml:space="preserve">
+    <value>Use SharedAccessProtocol.HttpsOnly when creating Shared Access Signature (SAS)</value>
```

Comment:
>Use SharedAccessProtocol.HttpsOnly when creating Shared Access Signature (SAS) [](start = 11, length = 78)

Consider using role-based access control instead of a Shared Access Signature (SAS) if possible.  If you still need to use a SAS, specify SharedAccessProtocol.HttpsOnly.